codereadyworkspaces/stack/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM registry.redhat.io/codeready-workspaces/plugin-java8-rhel8:2.10-4
+
+ENV ARGOCD_VERSION=2.0.5
+ENV YQ_VERSION=v4.11.1
+ENV HELM_VERSION=v3.6.0
+ENV KUBESEAL_VERSION=v0.13.1
+ENV TEKTON_VERSION=0.20.0
+ENV OC_VERSION=4.8.3
+ENV JQ_VERSION=1.6
+ENV CONFTEST_VERSION=0.23.0
+ENV ZSH_VERSION=5.8
+
+USER root
+
+# argo
+RUN curl -sL https://github.com/argoproj/argo-cd/releases/download/v${ARGOCD_VERSION}/argocd-linux-amd64 -o /usr/local/bin/argocd && \
+    chmod -R 775 /usr/local/bin/argocd && \
+    echo "🐙🐙🐙🐙🐙"
+
+#helm
+RUN  curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+     chmod 700 get_helm.sh && \
+     ./get_helm.sh --version ${HELM_VERSION} && \
```

Comment:
something funky here ... when i run (with ubi8 changes below) ... helm fails
```
virt:~/git/enablement-framework/codereadyworkspaces ⎇ main#1a20d50$ podman run --rm -it localhost/stack-do500:latest bash
warning: unable to access '/home/jboss/.config/git/config': Permission denied
[1001@4cb49ce667e3 projects]$ zsh
➜  /projects ls
get_helm.sh
➜  /projects 
➜  /projects helm version
WARNING: stat /home/jboss/.config/helm/registry.json: permission denied
/home/jboss/.config/helm/registry.json
github.com/deislabs/oras/pkg/auth/docker.NewClient
	github.com/deislabs/oras@v0.11.1/pkg/auth/docker/client.go:42
helm.sh/helm/v3/internal/experimental/registry.NewClient
	helm.sh/helm/v3/internal/experimental/registry/client.go:70
main.newRootCmd
	helm.sh/helm/v3/cmd/helm/root.go:156
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:70
runtime.main
	runtime/proc.go:225
runtime.goexit
	runtime/asm_amd64.s:1371
```
and may need to rm the get_helm.sh script huh ?